test/WitthdrawWorkflow.t.sol

```diff
@@ -0,0 +1,204 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.18;
+
+import "forge-std/Test.sol";
+import "forge-std/console.sol";
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import "@aa/core/EntryPoint.sol";
+
+import "../src/access/AccessRegistry.sol";
+import "../src/access/AccessPoint.sol";
+import "../src/access/workflows/WithdrawWorkflow.sol";
+import "../src/interfaces/IAccessPoint.sol";
+import "../src/interfaces/IKintoEntryPoint.sol";
+import "../src/paymasters/SignaturePaymaster.sol";
+
+import "./helpers/UserOp.sol";
+import "./helpers/ERC20Mock.sol";
+import "./helpers/UUPSProxy.sol";
+
+contract WithdrawWorkflowTest is UserOp {
+    using ECDSA for bytes32;
+
+    SignaturePaymaster paymaster;
+    IKintoEntryPoint entryPoint;
+    AccessRegistry internal accessRegistry;
+    IAccessPoint internal accessPoint;
+    WithdrawWorkflow internal withdrawWorkflow;
+    ERC20Mock internal token;
+
+    uint48 internal validUntil = 0xdeadbeef;
+    uint48 internal validAfter = 1234;
+
+    uint256 internal defaultAmount = 1e3 * 1e18;
+
+    function setUp() public {
+        vm.deal(_owner, 100 ether);
+        token = new ERC20Mock("Token", "TNK", 18);
+
+        entryPoint = IKintoEntryPoint(address(new EntryPoint{salt: 0}()));
```

Comment:
for my own understanding, on Kinto chain, we will need to deploy these contracts using the factory, correct? @rrecuero